registration/forms.py

```diff
@@ -320,6 +330,18 @@ def clean_password_confirmation(self):
             )
         return password
 
+    def clean_privacy_policy_url(self):
```

Comment:
The user should be able to submit the form with blank privacy and TOS fields, so that they take the default values.

Reply:
The approach I've taken allows the user to edit the mutable fields without requiring them to also add a privacy policy, or automatically adding one in the background that they haven't had an opportunity to review. It's a little more complex, but it keeps things visible and means existing instance owners can add the policy URL when they're ready. If we want to force them to have a privacy policy, we should notify them, give them a timeline and apply our default if they don't respond.

Reply:
I've removed this, and instead allowed empty values for both new and existing applications.